libobjrenderer/src/objectsscene.cpp

```diff
@@ -854,6 +854,14 @@ QGraphicsView *ObjectsScene::getActiveViewport(void)
 	return(view_p);
 }
 
+void ObjectsScene::adjustViewportToItems(QList<BaseObjectView *> items)
```

Comment:
@Maxzor what about moving this implementation to GQB plugin? Since it is only referenced from there it makes more sense to stay on the plugin's side. If, someday, there's the need to use such feature we can grab it back from plugin's code. What do you think?

Reply:
Sure why not!